var/httpd/htdocs/js/Core.AJAX.js

```diff
@@ -394,6 +394,12 @@ Core.AJAX = (function (TargetNS) {
                 UpdateTextarea($Element, DataValue);
                 return;
             }
+            
+            // check box elements
+            if ( $Element.is(':checkbox') ) {
+                $Element.prop('checked', false);
```

Comment:
Be careful, in the specific case DataValue will be 0, in which case this behaviour is correct, and I'm not aware that there would ever be a situation in which we will send a 1 here. Still you should at least check it, please. :)